### DISCUSSION TITLE
Adb logcat on sunday

### DISCUSSION THREAD

**Discussion by owenconnorz** [z-huang/music#1015] [Ideas]
On Sunday uk time I'll be doing tests with the logcat to find all the issues I'm facing and maybe many others to see if I can help you find any issues from

Login not showing email and username.

Artist section like albums,songs,singles not showing.

Any others I can find will be added to the list but unable to help others that are having issues I'm not facing hopefully I can help and find the main issues to help with the next release.

Thanks and all the best @z-huang

**Comment by z-huang** [comment]
The sooner the better. I'll release 0.5.3 before Sunday.

**Comment by owenconnorz** [comment]
Ahh OK well I can do it in the morning as iv been at work today but I'll get up early and do my best to get it over to u ASAP

**Comment by owenconnorz** [comment]
Hey @z-huang I'm having a hard time getting any info why this is happening on adb I'm searching through but can't find much info on the issues BUT can you maybe build a full release for me to test on my phone as the FULL Release shows albums songs and singles perfectly fine if so this would be very helpful 😊

**Reply by z-huang** [reply]
In this case, I'll release 0.5.3 directly.

**Reply by owenconnorz** [reply]
What about the song art can that be changed back to original as it doesn't look right at the moment

**Reply by z-huang** [reply]
Because the app can't find the song album, the song is recognized as a video. You can read this again: https://github.com/z-huang/InnerTune/issues/1013#issuecomment-1699320620.

**Reply by owenconnorz** [reply]
@z-huang I can confirm this is a issue with debug and not full release very confused on why 🤣🤣

**Reply by owenconnorz** [reply]
Oh no its something to do with backing up from debug to full release and it's caused it to happen again maybe the backup isn't just getting my songs it's getting everything on the app
